staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -601,7 +601,7 @@ message PodSandboxStatusResponse {
     map<string, string> info = 2;
     // Container statuses
     repeated ContainerStatus containers_statuses = 3;
-    // Timestamp at which container and pod statuses were recorded
+    // Timestamp in nanoseconds at which container and pod statuses were recorded
```

Comment:
this seems like it should be in a distinct commit from the readme update?

Reply:
right, fixed